src/number/NegativeFloat.ts

```diff
@@ -70,7 +73,7 @@ export type SchemableParams2C<S extends URIS2> = Kind2<S, unknown, NegativeFloat
  * @category Refinements
  */
 export const isNegativeFloat = (f: number): f is NegativeFloat =>
-  typeof f === 'number' && G.number.is(f) && f < 0 && f >= Number.MIN_SAFE_INTEGER
+  typeof f === 'number' && G.number.is(f) && f < 0 && f >= -Number.MAX_VALUE
```

Comment:
`MIN`/`MAX_SAFE_INTEGER` is nowhere close to the largest float that can be represented; copy/paste oversight perhaps?